### DISCUSSION TITLE
Code block in a list item

### DISCUSSION THREAD

**Discussion by erkanvatan** [nvim-neorg/neorg#724] [Q&A]
Hello,
I am trying to add a code block under a ordered list item but code block closes the list so new list items start from index 1. Is there a way to add code block and not close the paragraph?

Example:
~ First item
    @code bash
    echo "this belongs to first item"
    @end
~ Second item

**Comment by miraculusik** [comment]
can you try 
```
- First Item
  @code bash
  echo "this belongs to first item"
  @end
- Second Item
```
i think you skip spaces under list.
